packages/greenbox-data/data/items.ts

```diff
@@ -121,6 +121,7 @@ export const specialItems = [
   3138, // Hodgman's whackin' stick
   3139, // Hodgman's imaginary hamster
   3140, // old soft shoes
```

Comment:
```suggestion
```
this got duped somehow and is already in the Hobo Nickles section, probably my bad when i added them and copy / pasted wrong